.gitmodules

```diff
@@ -18,10 +18,10 @@
 	path = third-party/libutp
 	url = https://github.com/transmission/libutp
 	branch = post-3.4-transmission
-[submodule "third-party/miniupnpc"]
-	path = third-party/miniupnpc
-	url = https://github.com/transmission/miniupnpc
-	branch = post-2.0.20170509-transmission
+[submodule "third-party/miniupnp"]
+	path = third-party/miniupnp
+	url = https://github.com/transmission/miniupnp.git
+	branch = post-2.3.0
```

Comment:
If we aren't going to make any more local patches to the miniupnpc library, how about we just track the master branch?

I was confused for a minute when I saw we are tracking a commit newer than the branch head. 😅

Reply:
let's decide after we figure out the Windows build issue